README.md

```diff
@@ -12,7 +12,7 @@ From a technical point of view, the implementation of this game needs to be plat
 
 ![board](/img/snakesandladdersboard.jpg)
 
-The first feature we want implemented is the ability to move your token across the board using dice rolls. Players will need the ability to roll a dice, move their token the number of squares indicated by the dice roll and should win if they land on the final square. The feature is splitted in three user stories.
```

Comment:
The only features we want implemented are the ones explained in the user stories below. First is the ability to move your...(a partir de aquí, tal y como está)